docs/subsystems/realms.md

```diff
@@ -77,7 +77,7 @@ will try to get the page on your behalf. Since zulipdev.com points
 to 127.0.0.1 the proxy server is likely to give you a 503 error. The
 workaround is to disable your proxy for `*.zulipdev.com`. The DNS
 lookup should still work even if you disable proxy for
-*.zulipdev.com. If it doesn't you can add zulipdev.com records in
+\*.zulipdev.com. If it doesn't you can add zulipdev.com records in
```

Comment:
I think this should be in a code block instead of escaped (not important but just noticed while reading).

Reply:
I’m not sure hostnames are typically considered “code”, although certainly there’s a consistency issue to be resolved here with the other instances in this paragraph. :shrug:

Reply:
I like doing code blocks around fake hostnames like `foo.example.com`, which is sorta what this is, to prevent anything from trying to linkify them (and then our tooling from trying to verify the link isn't broken, etc.).  Seems like this is kinda in that category.

Reply:
[CommonMark](https://spec.commonmark.org/0.30/#autolinks) doesn't autolink without angle brackets, and even [GFM](https://github.github.com/gfm/#autolinks-extension-) doesn't autolink without https:// or www. The even more aggressive bare hostname autolinking is entirely our own extension that doesn't apply to RTD (and honestly, I don't think is useful). This is a typographical question.

Reply:
OK.  Probably fine to leave as-is.